inflation-inject-processor/src/main/java/com/squareup/inject/inflation/processor/InflationInjectProcessor.kt

```diff
@@ -43,6 +45,7 @@ import javax.lang.model.util.Types
 import javax.tools.Diagnostic.Kind.ERROR
 import javax.tools.Diagnostic.Kind.WARNING
 
+@IncrementalAnnotationProcessor(AGGREGATING)
```

Comment:
here i'm applying the same reasoning as here: https://github.com/runningcode/AssistedInject/blob/cb79d319150a6a07985aa5684b93e96e0551001e/assisted-inject-processor-dagger2/src/main/java/com/squareup/inject/assisted/dagger2/processor/AssistedInjectDagger2Processor.kt#L170,L173

Reply:
Maybe we should split this processor into two? The module one is definitely aggregating but the one which generate the factories is isolating.

PR welcome for that too, I guess.

Reply:
That'd be a nice follow-up